File: C017.cs
public class Hello{
    public static void Main(){
        // 自分の得意な言語で
        // Let's チャレンジ！！
        var parentNum = System.Console.ReadLine();
        string [] pN = parentNum.Split(new char[] {' '});
        int p1 = int.Parse(pN[0]);
        int p2 = int.Parse(pN[1]);
        
        var child = System.Console.ReadLine();
        int c = int.Parse(child);
        
        int counter;
        
        for(counter=0; counter<c; counter++)
        {
            var childNum = System.Console.ReadLine();
            string [] cN = childNum.Split(new char[] {' '});
            int c1 = int.Parse(cN[0]);
            int c2 = int.Parse(cN[1]);
            
            if(p1 > c1)
            {
                System.Console.WriteLine("High");
            }
            else if(p1 < c1)
            {
                System.Console.WriteLine("Low");
            }
            else
            {
                if(p2 < c2)
                {
                    System.Console.WriteLine("High");
                }
                else
                {
                    System.Console.WriteLine("Low");
                }
            }
        }
    }
}

//全パスできました！

File: c017.php
<?php
    //C017 ハイアンドロー
    //全パス済みです

    $par = fgets(STDIN);    //親の数字
    $parSep = explode(' ', $par);
    $childNum = fgets(STDIN); //子の数
    //子の数分繰り返す
    for($i = 0; $i < $childNum; $i++){
        //子の数字を配列に入れる
        $child[] = fgets(STDIN);
        //var_dump($child);
        //スペースで区切る
        $childSep = explode(" ", $child[$i]);
        //var_dump($childSep);
        for($j = 0; $j < 1; $j++){
            //print $childSep[$j];
            //print $parSep[$j];
            if($childSep[$j] < $parSep[$j]){
                echo "High\n";

            }elseif ($childSep[$j] == $parSep[$j] ) {
                $j = 1;
                if ($childSep[$j] > $parSep[$j]) {
                    echo "High\n";
                }else{
                    echo "Low\n";
                }
            }else{
                echo "Low\n";
            }
        }
    }
?>
